packages/psss-server/.env.example

```diff
@@ -1,3 +1,4 @@
+CLIENT_BASIC_AUTH_HEADER=
```

Comment:
What is this needed by? I can't seem to see any reference to it in the code...

Reply:
I had some issues running the server locally so I thought this was required for CORS, but probably it was another .env variable...

I see this is absent in the server so I'll remove it. Great pickup!